<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherClientMain</class>
 <widget class="QMainWindow" name="WeatherClientMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BEST WeatherStation Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>SMAP Server</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="outHost">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>192.168.1.108</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="outPort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>8080</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="inTestConnection">
         <property name="text">
          <string>Test Connection</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="inOpenPostgre">
         <property name="text">
          <string>Open Postgre</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>uuid</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextBrowser" name="outLogBrowser"/>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>UUID</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QDateTimeEdit" name="inStartTime"/>
        </item>
        <item row="1" column="3" rowspan="3">
         <widget class="QListView" name="inUUIDList"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>start</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>end</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateTimeEdit" name="inEndTime"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Plot</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>uuid</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>pushButton</zorder>
       <zorder>groupBox</zorder>
       <zorder>label_4</zorder>
       <zorder>label_5</zorder>
       <zorder>inStartTime</zorder>
       <zorder>inEndTime</zorder>
       <zorder>inUUIDList</zorder>
       <zorder>label_6</zorder>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="mplWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfig_Editor"/>
    <addaction name="actionDatabase_Admin"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfig_Editor">
   <property name="text">
    <string>Config Editor</string>
   </property>
  </action>
  <action name="actionDatabase_Admin">
   <property name="text">
    <string>Database Admin</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
